### ISSUE TITLE
Internal server error

### ISSUE THREAD

**Issue by Tanaya9299** [afids/afids-validator#91]
## The problem

I tested the afids protocol and while using the validator I am getting internal error

## Environment

* Safari (13.1.1)

_

**Comment by kaitj** [comment]
Hi there! Could you let us know which dropdown option was selected as well as an example of the .fcsv that was used?

**Comment by Tanaya9299** [comment]
[2009bMNI_T1_TM_01.txt](https://github.com/afids/afids-validator/files/4772594/2009bMNI_T1_TM_01.txt)

**Comment by kaitj** [comment]
@Tanaya9299 - Thank you. I took a look at the contents of the .txt file. It seems as if there is some mismatch/missing of information in the first two fiducials (AC, PC) which is causing the internal error. I've included below the two lines that are causing this issue.

Original:
```
vtkMRMLMarkupsFiducialNode_0,0.023,-8.165,-16.415,0.000,0.000,0.000,1.000,1,1,1,AC,,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_1,0.536,-25.470,-2.099,0.000,0.000,0.000,1.000,1,1,1,PC,,vtkMRMLScalarVolumeNode1
```

Replace with:
```
vtkMRMLMarkupsFiducialNode_0,0.023,-8.165,-16.415,0.000,0.000,0.000,1.000,1,1,1,1,AC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_1,0.536,-25.470,-2.099,0.000,0.000,0.000,1.000,1,1,1,2,PC,vtkMRMLScalarVolumeNode1
```

Note the two commas following label description (AC, PC) and the missing label (1, 2). This should fix the file to get around the file.

@afids/afids-validator - note to discuss implementation of checking mismatched column contents to columns. This error bypasses current checks and throws an internal error - also replicated on Chrome (83.0.4103.97). There are correct number of columns, however label description (`AC`) was in the place of label, and description ended up blank.
